bin/configurator/static/_includes/ru/configurator/tab/ci/argocd-with-gitlab-ci-cd/simple/docker-runner/linux/buildah/infra.md.liquid

```diff
@@ -0,0 +1,62 @@
+## Требования
+
+- GitLab;
+
+- Хост для запуска GitLab Runner'а с предустановленным:
+
+  - [Docker Engine](https://docs.docker.com/engine/install/);
+
+- [ArgoCD](https://argo-cd.readthedocs.io/en/stable/getting_started/#1-install-argo-cd).
+
+## Установка GitLab Runner
+
+Установите GitLab Runner на выделенный для него хост, следуя [официальным инструкциям](https://docs.gitlab.com/runner/install/linux-repository.html).
+
+## Регистрация GitLab Runner
+
+Следуйте [официальным инструкциям](https://docs.gitlab.com/runner/register/index.html) по регистрации GitLab Runner'а в GitLab'е; укажите `docker` в качестве исполнителя и любой образ в качестве default image (например, `alpine`).
+
+## Настройка GitLab Runner
+
+На хосте с GitLab Runner'ом откройте его конфигурационный файл `config.toml` и добавьте следующие параметры к GitLab Runner'у, который вы зарегистрировали выше:
+
+```toml
+[[runners]]
+  name = "<имя зарегистрированного Runner'а>"
+  [runners.docker]
+    security_opt = ["seccomp:unconfined", "apparmor:unconfined"]
+    volumes = ["/home/build/.werf"]
+```
+
+Если версия ядра Linux на хосте с GitLab Runner'ом ≤ 5.12, установите `fuse` на хост и добавьте дополнительный параметр в файл `config.toml`:
+
+```toml
+[[runners]]
+  name = "<имя зарегистрированного Runner'а>"
+  [runners.docker]
+    devices = ["/dev/fuse"]
+```
+
+При необходимости проведите [дополнительную настройку](https://docs.gitlab.com/runner/configuration/advanced-configuration.html) GitLab Runner'а.
+
+## Настройка реестра контейнеров
+
+[Включите сборку мусора](https://docs.gitlab.com/ee/administration/packages/container_registry.html#container-registry-garbage-collection) в своем реестре контейнеров.
```

Comment:
> реестре контейнеров

It seems like we previously decided to write "container registry" everywhere? @alexey-igrychev

Reply:
Let's use "container registry" for any language.

Reply:
```suggestion
[Включите сборку мусора](https://docs.gitlab.com/ee/administration/packages/container_registry.html#container-registry-garbage-collection) в своем container registry.
```